res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 
 .mx_RoomHeader_info {
     flex: 1;
+    /* Spacing of 64px between the topic and the buttons, but we have
+       to account for the gap of the flex items */
+    padding-right: var(--cpd-space-13x);
```

Comment:
![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/6fa23a75-1b47-4f30-82b0-bd936629da92)

Based on the comment it looks like the wrong variable is used

Reply:
The gap on the flex parent is `--cpd-space-3x` which is `12px`.  It all adds up to 64px. Might cahnge the owrding of the comment